packages/core/src/types/displayArea.ts

```diff
@@ -0,0 +1,16 @@
+type InitialDisplayArea = {
+  areaX: number;
+  areaY: number;
+};
+
+type ImageFocalPoint = {
+  focalX: number;
+  focalY: number;
+};
+
+type DisplayArea = {
+  imageArea: InitialDisplayArea;
+  imageFocalPoint: ImageFocalPoint;
```

Comment:
This doesn't quite work without a second focal point -  you need to be able to handle the following cases:
1. Center of the image as a center point - WITHOUT knowing about the size of the display area OR the size of the image
2. Left image aligned or Right image aligned
3. Zoomed to specific point in image as center of displayed area.

For #1, using fractional points as the DICOMweb requests do for displayed area makes sense.
For #2, having both an image focal point and a display focal point makes sense, where the display focal point is optional, and default to the center of the image area.
For #3, using both 1,2 you get exactly what you want.

So, what you need is to have the initial display area as a fraction between 0 and 1 (but do NOT constrain it to 1 at the top end, but strictly larger than 0 at the bottom end - in fact larger than 1% makes sense)

The ImageFocalPoint should be renamed FocalPoint, and there should be two of them (could also call it ViewFocalPoint, or if you leave it ImageFocalPoint, then realize it is image area related).
Change DisplayArea to be:
// The focus point of interest for this image
imageFocalPoint?: ImageFocalPoint
// The focus point of interest for this display area.  Defaults if undefined to the center of the display area
displayFocalPoint?: ImageFocalPoint

You might also want to have a units setting - so that you can provide exact zoom levels - the three units of interest would specify the display area in any of:
Scale to Fit -  as a fraction of the original image size
Zoom - as a zoom scale (independent x,y) - 1.0 means 1 image pixel displays as 1 screen pixel
TrueSize - as a multiple of the true size, displayed on screen, eg 1.0 means 1 mm in body displays as 1 mm on screen

AND, either areaX or areaY should be nullable so that you can specify one exactly, without consideration for how the OTHER one fits.

I'm well aware this is much harder, but it does a lot of good for how to apply settings consistently, before the image area is actually known.   If you KNOW the actual sizes of images, it is easy to set them in fractions, but if you don't know, then there isn't a good way to know what values to use up front.